### ISSUE TITLE
Stellarium 0.20.2 freezed at start on Win10

### ISSUE THREAD

**Issue by bemo47** [Stellarium/stellarium#1142]
### Actual Behaviour
Describe or maybe attach a screenshot?

### System
* Stellarium version: <Name of downloaded installable file?>
* Operating system: <Name, version number>
* Graphics Card: <Manufacturer (likely Intel, NVidia, AMD?), Model (HD, Geforce, Radeon..., with model number), driver version?>
* Screen type (if applicable): Resolution, HighDPI, scaling

### Logfile 
If possible, attach the logfile `log.txt` from your user data directory. Look into the Guide for its location.

**Comment by bemo47** [comment]
I installed the last Stellarium version, 0.20.2 on my Windows 10 1909

When i activate the plugin Telescope Control, after restart of Stellarium, it opens full screen as usual but i cannot do anything on the menus.

Everything is freezed.

I did uninstall Stellarium, and install it again but its same.

Regards

Bernard

**Comment by bemo47** [comment]
oh, sorry, but now, after about 1 hour, i tried again and it works normally ....
I really don't understand why i had these freeze but now it seams ok.

**Comment by bemo47** [comment]
and sorry again, but after some trials without issues, now it start again to freeze.....

**Comment by gzotti** [comment]
Quoting from the report template which you apparently preferred to ignore:

> **Logfile**
> 
> If possible, attach the logfile log.txt from your user data directory. Look into the Guide for its location.

**Comment by bemo47** [comment]
logfile is attached in the first message

**Comment by bemo47** [comment]
but i attach it again here
[log.txt](https://github.com/Stellarium/stellarium/files/4831633/log.txt)

**Comment by gzotti** [comment]
These are from successful runs (unloading plugin modules at shutdown...)

**Comment by bemo47** [comment]
yes but i can't do anything on stellarium, its freezed, when i move my cursor i cannot see the menus, the screen doesn't move, so at the end i do a close of the windows throught the alt+tab function

**Comment by gzotti** [comment]
When Stellarium is frozen and you kill it, the logfile should look different.

You can try ANGLE or MESA modes to see if it's a graphics problem.

**Comment by bemo47** [comment]
here again, new trial just now, stellarium opens, full screen but totally freezed after that.

[log.txt](https://github.com/Stellarium/stellarium/files/4831694/log.txt)

**Comment by bemo47** [comment]
ok, tried with ANGLE and it worked, what can be the conclusion ?
should i work with ANGLE ?

**Comment by bemo47** [comment]
MESA working also

**Comment by gzotti** [comment]
Was this a fresh install or did you update? Maybe delete/rename config.ini to start with fresh defaults.
I cannot say what may not work with OpenGL, but if ANGLE works, then just use it.

**Comment by bemo47** [comment]
its an update from previous version of Stellarium but i did an unisntall and asked to remove all files before installing the new 0.20.2 version.
but i can go to my disk and check if the config.ini has been deleted too.
anyway, with the ANGLE or MESA it will work well also ?

**Comment by gzotti** [comment]
ANGLE should be almost as fast, it just translates OpenGL to DirectX. MESA on Windows runs on the CPU only, so this is much slower, but a last fallback if the GPU is bad. But a Geforce with current drivers should usually be fine.

**Comment by bemo47** [comment]
ok, thanks, but what i noticed is that it freeze only after activation of the telescope control plugin. When i open Stellarium at first its ok, then i select and activate the plugin, then restart, and now its freezed.

**Comment by gzotti** [comment]
OK. Sorry, but freezing telescopes are not my territory. At least this is some indication what may fail.

**Comment by bemo47** [comment]
ok, anyway thanks for having found a bypass :)
and now i can launch with the normal stellarium and it works !!! really strange...

**Comment by alex-w** [comment]
Are you defined to auto connect to the mount in the Telescope Control plugin and tried startup planetarium without connection of hardware?

**Comment by bemo47** [comment]
no auto connect, anyway it seems that i can't click this option, but i don't need

![image](https://user-images.githubusercontent.com/44326240/85758991-1867bc00-b711-11ea-8b02-e69519fd23b6.png)

**Comment by ManyExposures** [comment]
> no auto connect, anyway it seems that i can't click this option, but i don't need
> 
> ![image](https://user-images.githubusercontent.com/44326240/85758991-1867bc00-b711-11ea-8b02-e69519fd23b6.png)

I have the same issue. No matter what I try (OpenGL or MESA) the auto-connect at startup is greyed out. Also, at startup, the scope/mount circle stops somewhere on the east and only comes to the celestial pole if I park and unpark EQMOD. This just started to happen with this 20.2 release.

**Comment by gzotti** [comment]
This is not OpenGL vs. ANGLE. 

Finally I see ASCOM selected. Maybe @gionkunz can help you? What was the version of Stellarium that worked for you before?

**Comment by gionkunz** [comment]
I know there was an issue with ASCOM telescopes automatically starting on Stellarium start when there was no gear connected. Was disabling the auto start feature when ASCOM was selected as a quick fix / workaround to prevent that from happening? Maybe there's something wrong there and the telescope is set to automatically start even if the checkbox is not ticked and disabled?

@ManyExposures @bemo47 Can you post your setting files? @gzotti do you recollect the location on Windows where the telescope configuration is written to?

**Comment by gzotti** [comment]
That should be $USER_DATA\modules\TelescopeControl\telescopes.json
Some flags also in $USER_DATA\config.ini, but these probably are not important here.
I am not really familiar with the TC plugin code. You have certainly already gained more insight.

**Comment by ManyExposures** [comment]
> I know there was an issue with ASCOM telescopes automatically starting on Stellarium start when there was no gear connected. Was disabling the auto start feature when ASCOM was selected as a quick fix / workaround to prevent that from happening? Maybe there's something wrong there and the telescope is set to automatically start even if the checkbox is not ticked and disabled?
> 
> @ManyExposures @bemo47 Can you post your setting files? @gzotti do you recollect the location on Windows where the telescope configuration is written to?

@gionkunz  thanks for the response. Last night I worked a little longer on this and found the cause of the problem. It was on EQMOD. I ran ASCOM diagnostics and one of the checks for EQMOD failed. I googled the error and an old post on Sratgazers Lounge gave me the clue. The trick was to delete all the EQMOD .ini files. Did that, re-register EQMOD, put my location in, and found the COM port and all good now. The checkbox for open on startup is available and stays checked.

What caused the problem with EQMOD? My guess is that when I was playing with the Park script in APT the other night it may have stored a park point out there on the east where I was at the moment. This is just a guess.

My issue was not related to Stellarium 20.2.

Thank you for offering to help. 

Clear skies

**Comment by alex-w** [comment]
Hmmm... Could you install version 0.20.1 into separate directory and check behavior of plugin with automatic connection mode?

**Comment by ManyExposures** [comment]
Alex. I tried to go back to version 20.1 that I had no issues with before and the same behaviuor as 20.2. That is when I realized the problem was not with Stellarium. 

So I did this as posted here

https://github.com/Stellarium/stellarium/issues/1142#issuecomment-654806173

**Comment by gionkunz** [comment]
Okay, @ManyExposures , glad you found the issue :-) Let's wait for a reply of the OP @bemo47 to see if there's a similar issue

**Comment by alex-w** [comment]
@gionkunz could you check an issue with ASCOM telescopes automatically starting on Stellarium start when there was no gear connected? I mean removing my hack to disabling autoconnecting.

**Comment by gionkunz** [comment]
Hi @alex-w , ah so that is actually put into place :-) Thanks for clarifying! Makes sense to disable auto-connect for ASCOM as long as the bug is there as a short term measure! 

I'll need to see when I find time for this, hopefully sometime this week. Will keep you guys updated.

Cheers
Gion

**Comment by gzotti** [comment]
@gionkunz @alex-w what's the state here? Do we need to keep it open?

**Comment by alex-w** [comment]
> @gionkunz @alex-w what's the state here? Do we need to keep it open?

This is sad, but yes - we need to keep it open until classical ASCOM is supported by Stellarium.
